internal/journal/voljournal.go

```diff
@@ -691,9 +691,22 @@ func (conn *Connection) GetImageAttributes(ctx context.Context, pool, objectUUID
 }
 
 // StoreImageID stores the image ID in omap.
-func (conn *Connection) StoreImageID(ctx context.Context, pool, reservedUUID, imageID string) error {
-	err := setOMapKeys(ctx, conn, pool, conn.config.namespace, conn.config.cephUUIDDirectoryPrefix+reservedUUID,
-		map[string]string{conn.config.csiImageIDKey: imageID})
+func (conn *Connection) StoreImageID(ctx context.Context, pool, reservedUUID, imageID, kmsID, owner string) error {
```

Comment:
the function is doing more work than storing the imageID. can we have a separate function to store the encryption ID and owner ?

Reply:
Maybe this change is not needed, as `ReserveName()` should set the correct metadata already. I'll try dropping this change and see how it goes.

Reply:
Yes, correct. This is for post metadata reservation, Once the Reservation is done, we need to reserve the imageID at last after creating the image.

Reply:
Dropping this does not seem to affect the functioning, so that looks good :+1: This change will not be included in the next update.